runtime/validate.go

```diff
@@ -152,3 +152,17 @@ func safeSQLName(name string) string {
 	}
 	return fmt.Sprintf("\"%s\"", strings.ReplaceAll(name, "\"", "\"\""))
 }
+
+func fullyQualifiedTableName(t *drivers.Table) string {
```

Comment:
Align naming with the other place this function exists. Actually, consider using `olap.Dialect().EscapeTable(t.Database, t.DatabaseSchema, t.Name)`.